contrib/debian/netdata-plugin-perf.postinst

```diff
@@ -7,7 +7,7 @@ case "$1" in
     chown root:netdata /usr/libexec/netdata/plugins.d/perf.plugin
     chmod 0750 /usr/libexec/netdata/plugins.d/perf.plugin
 
-    if ! setcap cap_perfmon+ep /usr/libexec/netdata/plugins.d/perf.plugin; then
+    if setcap cap_perfmon+ep /usr/libexec/netdata/plugins.d/perf.plugin; then
```

Comment:
Guys, do we need to 2 > /dev/null here? To suppress error output 

if cap doesn't exist you get this

```
fatal error: Invalid argument
usage: setcap [-h] [-q] [-v] [-n <rootid>] (-r|-|<caps>) <filename> [ ... (-r|-|<capsN>) <filenameN> ]

 Note <filename> must be a regular (non-symlink) file.
 -r          remove capability from file
 -           read capability text from stdin
 <capsN>     cap_from_text(3) formatted file capability
 [ Note: capsh --suggest="something..." might help you pick. ]
 -h          this message and exit status 0
 -q          quietly
 -v          validate supplied capability matches file
 -n <rootid> write a user namespace (!= 0) limited capability
 --license   display the license info
```

Reply:
Yes, we should be redirecting stderr to `/dev/null`